lib/response/set/setRequestCycle.js

```diff
@@ -17,7 +17,7 @@ module.exports = function setRequestCycle(model, observer, groups,
     var requestedPaths = requestedAndOptimizedPaths.requestedPaths;
 
     // we have exceeded the maximum retry limit.
-    if (count === model._maxRetries) {
+    if (count > model._maxRetries) {
```

Comment:
Just asking considering the original comparison logic, should this be `>=`?

Reply:
Yeah, that'd be safer. I made the change to align the retry counts with `getRequestCycle`; one started at 0, the other at 1. Starting at 1 lines up better with the Zuul retry header value.